### PR TITLE
Updates to performance and SEO

### PR DESCRIPTION
**Pull request by matthewfowles**
A Start on improving performance, lighthouse scores and SEO. This is not to be merged immediately and added too over time.

will add more info in: #199

### CODE REVIEW

**Comment by peduarte** [comment]
Deploy preview for *fathomlondon* ready!

Built with commit 1d2bfdf0c3419167065723de07bcd7f00318014d

https://deploy-preview-319--fathomlondon.netlify.com

File: src/components/ContactCard/index.js [hunk 1]
```diff
@@ -9,11 +9,19 @@ export function ContactCard({ email, tel, className }) {
...
+					aria-label="Contact Email"
+					href={`mailto:${email}`}
+					className="hover-white"
+				>
 					{email}
 				</a>
 				<br />
-				<a href={`tel:${tel}`} className="hover-white">
+				<a
+					aria-label="Contact Phon"
```
Comment (petecorreia) [inline src/components/ContactCard/index.js]: Typo here? Guessing it should be "Phone"

File: src/components/Footer/index.js [hunk 2]
```diff
@@ -52,6 +58,8 @@ export function Footer({ email, tel }) {
 								target="_blank"
 								rel="noopener noreferrer"
 								className="db w2 light-grey hover-white"
+								title="Fathom London Linked In"
+								aria-label="Fathom London Linked In"
```
Comment (petecorreia) [inline src/components/Footer/index.js]: Typos here on both attrs — should be "LinkedIn" (no space)

File: src/components/PostBody/index.js [hunk 3]
```diff
@@ -129,6 +130,8 @@ function SocialShare({ title, excerpt, baseUrl, slug, twitter }) {
 						href={`https://www.linkedin.com/shareArticle?mini=true&url=${baseUrl}${slug}?blog&title=${title}&summary=${excerpt}&source=${baseUrl}${slug}`}
 						rel="nofollow noopener noreferrer"
 						target="_blank"
+						title="Share on Linked In"
+						aria-label="Share on Linked In"
```
Comment (petecorreia) [inline src/components/PostBody/index.js]: Typos here on both attrs — should be "LinkedIn" (no space)

File: gatsby-config.js [hunk 4]
```diff
@@ -35,5 +45,45 @@ module.exports = {
...
+				background_color: '#ffffff',
+				theme_color: '#ffffff',
+				display: 'standalone',
+				dir: 'ltr',
+				lang: 'en',
+				icon: 'src/assets/images/icon.png',
+				orientation: 'portrait',
+			},
+		},
+		'gatsby-plugin-offline',
```
Comment (petecorreia) [inline gatsby-config.js]: Let's have @peduarte review this bit

**Comment by petecorreia** [comment]
When navigating to article from homepage, the following happens (see gif):

bug

**Comment by matthewfowles** [comment]
@petecorreia I agree that is happening, The bit I don't get is it is either on the live site!!!!! So it's either the update to V2 or been there a while, Also the site is slow just on blog posts. I'm going to look into this!

File: src/components/PostBody/index.js (hunk 3 above)
Comment (matthewfowles) [reply src/components/PostBody/index.js]: Ok I really don't say it that way :)

**Comment by petecorreia** [comment]
@matthewfowles thanks for looking into it! had no idea if it was related to this PR 😄

**Comment by peduarte** [comment]
Does the CMS still work? I just checked on this branch and I get this:

image

**Pull request merged**